src/app.js

```diff
@@ -1 +1,90 @@
+/* eslint-disable */
 'use strict';
+/* eslint-enable */
+
+import inquirer from 'inquirer';
+import fs from 'fs/promises';
+
+const BullsAndCows = async() => {
+  const secret = (Math.random().toString().slice(-4));
```

Comment:
It's not the number with all different numbers. Check task instructions carefully.